app/controllers/games_controller.rb

```diff
@@ -5,6 +5,7 @@ class GamesController < ApplicationController
 
   def index
     authorize(Game)
+    @game = Game.top10
```

Comment:
hay una convención no escrita en que los que son enumeradles se manejan en plural
```suggestion
    @games = Game.top(10)
```